BuildLoader.py

```diff
@@ -744,6 +761,10 @@ def create_platform_vars (self):
 			if acm_base & 0x7FFF:
 				raise Exception ('ACM base[FSP-T+CAR:0x%x] must be 32KB aligned!' % acm_base)
 
+		if getattr(self._board, 'ACM3_SIZE') > 0:
+			acm3_base = getattr(self._board, 'ACM3_BASE')
+			if acm3_base & 0xFF:
+				raise Exception ('ACM3 base[FSP-T+CAR:0x%x] must be 4KB aligned!' % acm3_base)
```

Comment:
from comment, ACM3 is 4KB aligned. should line 765 be changed to "if acm3_base & 0x0FFF:"?

Reply:
Good catch. Its a typo. Will change it.